dev/com.ibm.ws.ejbcontainer.mdb/src/com/ibm/ws/ejbcontainer/mdb/internal/MessageEndpointFactoryImpl.java

```diff
@@ -96,15 +101,14 @@ public MessageEndpointFactoryImpl() throws RemoteException {
     @Override
     public void initialize(EJSContainer ejsContainer,
                            BeanId id,
-                           BeanMetaData bmd)
-                    throws RemoteException
-    {
+                           BeanMetaData bmd) throws RemoteException {
         super.initialize(ejsContainer, id, bmd);
 
         // Get the activation spec name. This will have been obtained from the
         // binding file, or defaulted to "[app/]module/bean" conforming to the
         // java:global rules for binding an EJB.
         activationSvcId = bmd.ivActivationSpecJndiName;
+        name = "messageEndpoint-" + activationSvcId;
```

Comment:
This seems like an odd name;  I don't see the design for plausable component; should "messageEndpoint" be translated? Will the customer be more likely to pause an MDB or an ActivationSpec?  If MDB, then it seems the name should be more like app/module/bean (info can be obtained from bnd.getJ2EEName().  If it is possible to associate multiple MDBs with the same ActivationSpec, then this should probably be MDB info.   Is this name format consistent with other PausableComponent names?  Should it really start with a lowercase, or should it reflect the interface the component implments, so "MessageEndpoint"?